ores/scoring_systems/tests/util.py

```diff
@@ -34,8 +35,9 @@ def score(self, feature_values):
 
 class FakeExtractor(Extractor):
 
-    def extract(self, rev_ids, features, *args, **kwargs):
-        return [(None, [0 for feature in features]) for rev_id in rev_ids]
+    def extract(self, rev_ids, features, *args, caches={}, **kwargs):
+        return [(None, list(solve(features, cache=caches.get(rev_id, {}))))
+                for rev_id in rev_ids]
```

Comment:
The old version here was doing literally nothing.  We need to it abide by the extraction pattern for the test to pass.